h/static/styles/annotation-thread.scss

```diff
@@ -0,0 +1,72 @@
+.annotation-thread {
+  display: flex;
+  flex-direction: row;
+}
+
+// Direct or nested reply to an annotation
+.annotation-thread--reply {
+  // The negative margin is used so that the hovered background of reply threads
```

Comment:
Note: This needs to be removed and simplified as we decided not to implement the hovered background from the specs. The comment is also out of date here.